pkg/plugin/plugin_test.go

```diff
@@ -91,6 +91,15 @@ func TestEncrypt(t *testing.T) {
 			healthErr: true,
 			checkErr:  true,
 		},
+		{
+			input:     plainMessage,
+			ctx:       nil,
+			output:    "",
+			err:       awserr.New("AccessDeniedException", "The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access", errors.New("fail")),
```

Comment:
For completeness, can you add a test case for AccessDenied that is not user induced?

Reply:
> For completeness, can you add a test case for AccessDenied that is not user induced?

@micahhausler Added a test for the `AccessDenied` that is not user induced